src/webview/MessageList.js

```diff
@@ -199,6 +199,8 @@ class MessageList extends Component<Props> {
     return false;
   };
 
+  getBlankWebView = () => <View style={this.context.styles.webview} />;
```

Comment:
This name doesn't feel quite right -- after all, what this provides isn't a webview.

I think having the name start with "render" would also be helpful, as it's basically another render method: a method whose job is to prepare and return a bunch of components to get put into the tree.

In fact `renderLoading` (same name as the prop it's passed to) would be pretty good.

Reply:
Sure, `renderLoading` sounds good to me.

I'm not sure I'm entirely on board with react-native-webview's naming choices; as far as I can tell, `renderLoading` is only called to render the _initial_ load, not any subsequent ones, and is entirely irrelevant to something like `onShouldStartLoadWithRequest`, which we use. But at least we can be consistent with the names they've chosen (as, indeed, we've done with `onShouldStartLoadWithRequest).